### PR TITLE
feat-input

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (82)

### CODE REVIEW

File: comp-layouts/input.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          Math<br>
+          <input type="checkbox" value="English">
+          English<br>
+          <input type="checkbox" value="Science">
+          Science<br>
+     </form>
+     <hr><br>
+     <form>
+          <input type="button" value="Button" onclick="alert('hello world')">
+     </form>
```
Comment (Sameony) [inline comp-layouts/input.html:39]: you dont need to have a form tag to implement the input tag

File: stylesheets/components/input.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import url(/stylesheets/main.css);
+
+.input {
+  position: relative;
+  left: 20px;
```
Comment (Sameony) [inline stylesheets/components/input.css]: if you are only trying to move this element from the left then maybe you could have used margin left.

**Review by Sameony — COMMENTED** [review]
a few adjustments, do pick those up

File: comp-layouts/input.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layouts/input.html:39]: yes but if someone try to run button so for that i given ( its extra i added).

File: stylesheets/components/input.css (hunk 2 above)
Comment (iamsaurav28) [reply stylesheets/components/input.css]: okay done

**Review by Sameony — APPROVED** [review]

**Review by rahulcs754 — COMMENTED** [review]
I looked at your HTML code that is not good please try to prettier and another tool that will solve your HTML issue.

**Review by rahulcs754 — APPROVED** [review]
Always try to clean your code

**Pull request merged**